pkg/watchmanager/config.go

```diff
@@ -0,0 +1,32 @@
+package watchmanager
```

Comment:
We have a config package, should this be there?

Reply:
Wanted to keep the config related to watch manager completely separate and independent of other SDK package(some still used for harvester client though) to keep this as independent component.